packages/components/page-header/src/page-header.ts

```diff
@@ -1,9 +1,10 @@
-import type { ExtractPropTypes } from 'vue'
+import { Back } from '@element-plus/icons'
+import type { ExtractPropTypes, PropType, Component } from 'vue'
 
 export const pageHeaderProps = {
   icon: {
-    type: String,
-    default: 'el-icon-back',
+    type: [String, Object] as PropType<string | Component>,
```

Comment:
Use `buildProp` please.

Reply:
We can update it in the new PR

Reply:
Let's make a follow up PR for this then.